src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Controls/UnoWebView/UnoWebView.cs

```diff
@@ -127,8 +127,12 @@ private void OnNavigationStarting(WebView sender, WebViewNavigationStartingEvent
 				}
 			}
 #if __IOS__
-			// Needed as webview keeps for unknown reason the token, delete cookies each time avoid this behavior
-			ClearCacheAndCookies();
+			//Only apply clear cache when user logged out once (fixes bug for accept microsoft condition loop login)
```

Comment:
So the bug is still present if we login for the first time with a different account?

Reply:
I was told the problem is happening on first login with Uado on an account.

Reply:
Yes. But we implemented **### systematic** ClearCacheAndCookies() call to avoid a previous bug that was causing a loop to login page with a automatic login (token still here) when logging out. So the solution proposed here is to ClearCacheAndCookies only when a first logout has been done (as we want to keep CacheAndCookie for the user to be able to accept conditions at his first login). 

_"Needed as webview keeps for unknown reason the token, delete cookies each time avoid this behavior"_